widgets/src/main/java/org/pentaho/gwt/widgets/client/listbox/CustomListBox.java

```diff
@@ -1025,10 +1028,14 @@ public void onKeyUp( Widget widget, char c, int i ) {
           }
         }
         break;
-      case 27: // ESC
+      case 27: // Esc
+        break;
```

Comment:
By adding the new code, this _case_ is not needed then, here.

Reply:
Okay, will be redacting the "Escape" case